Sprint-3/revise/implement/password-validator.test.js

```diff
@@ -14,3 +14,31 @@ To be valid, a password must:
 
 You must breakdown this problem in order to solve it. Find one test case first and get that working
 */
+const passwordValidation = require("./password-validator");
+describe("passwordValidation", () => {
+  let existingPasswords = ["password123", "Qwerty1!"]; // Example existing passwords
+
+  it("should return false if password is already in existingPasswords list", () => {
+    expect(passwordValidation("password123")).toBe(false); // Password already exists
+  });
+
+  it("should return false if password is less than or equal to 5 characters", () => {
+    expect(passwordValidation("abc12")).toBe(false); // Too short
```

Comment:
`abc12`: this password is not valid for another reason, and not because it has less than 5 characters.